### PR TITLE
Change misleading comment in test.template.properties #3037

### LINKED ISSUE #3037: Change misleading comment in test.template.properties
On line 40, it says

<code># The values given below will work for the dev server.</code>

When in fact, the default values need to be changed for GodModeTest to pass.

### PR DESCRIPTION
**Pull request by julian1990**
Fixes #3037

### CODE REVIEW

**Comment by LowWeiLin** [comment]
The issue is to change the comment, not change the accounts.
(Also, there is no need to change test.admin.account)

**Comment by julian1990** [comment]
@LowWeiLin So I just need change the comment like this:
Line 40: # The values given below will work for the dev server (except GodModTest).

Should I add some comment on how to pass the GodModTest below this line? like:
#To pass GodModeTest, change ALL the accounts by appending to the back of it e.g. alice.tmms -> alice.tmms.asd

File: src/test/resources/test.template.properties [hunk 1]
```diff
@@ -37,7 +37,9 @@ test.firefox.path=
 
 ###############################################################################
 # Given below are the test accounts/passwords used for testing. 
-# The values given below will work for the dev server.
+# The values given below will work for the dev server(except one test case GodModTest).
+# To pass GodModTest, change ALL the accounts(test.admin.account) by appending to the back of it.
```
Comment (kanghj) [inline src/test/resources/test.template.properties]: Isn't this supposed to be "... change ALL the accounts by appending ..." without mentioning "(test.admin.account)"?

Comment (julian1990) [inline src/test/resources/test.template.properties]: Oh, I missed 'EXCEPT' when I typed this comment. :-(

**Pull request merged**